### PR TITLE
styling added to body element

### PR DESCRIPTION
**Pull request by warwickandrew**
fixes animation issue on flipcard within edge

### CODE REVIEW

File: less/flipcard.less [hunk 1]
```diff
@@ -28,6 +28,15 @@
 	transition: opacity @duration @delay !important;
 }
 
+  // ADAPT-4343 Fixes a display bug in Edge
+  // Introduces other bugs in IOS so make it exclusive to Edge
+  // ADAPT-5242 fixes animation issue on edge
+  html.edge {
+    body {
```
Comment (nilslp) [inline less/flipcard.less]: Probably no need to have these nested `html.edge body`

**Review by nilslp — CHANGES_REQUESTED** [review]

**Comment by nilslp** [comment]
Then bump the version of the plugin, Thanks

File: less/flipcard.less [hunk 2]
```diff
@@ -159,7 +166,7 @@
           // desktop
           padding: 20px 60px;
         }
-
+        
         .ie8 &,
```
Comment (cahirodoherty-learningpool) [inline less/flipcard.less:170]: I know it's outside the scope of the task, but IE8/9 styling can be removed here

**Review by cahirodoherty-learningpool — DISMISSED** [review]

File: less/flipcard.less (hunk 2 above)
Comment (nilslp) [reply less/flipcard.less:170]: I'm happy either way here, as we are (going to be) tracking all plugins legacy code and refactor so might be better to keep this PR simple.

Comment (warwickandrew) [reply less/flipcard.less:170]: Will leave it and add to the epic for plugins re-factor. Then do it all together.

File: less/flipcard.less [hunk 3]
```diff
@@ -159,7 +166,7 @@
           // desktop
           padding: 20px 60px;
         }
-
+
```
Comment (nilslp) [inline less/flipcard.less:169]: Spaces added here?

File: bower.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
     "name": "adapt-contrib-flipcard",
     "framework": ">=3.0.0",
-    "version": "3.0.0",
-    "homepage": "https://github.com/learningpool/adapt-contrib-flipcard",
-    "issues": "https://github.com/learningpool/adapt-contrib-flipcard/issues",
-    "repository": "git://github.com/learningpool/adapt-contrib-flipcard.git",
+    "version": "2.1.9",
+    "homepage": "https://github.com/ExultCorp/adapt-contrib-flipcard",
```
Comment (cahirodoherty-learningpool) [inline bower.json]: This has been mistakenly added from conflict resolution

**Review by cahirodoherty-learningpool — CHANGES_REQUESTED** [review]

**Review by nilslp — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]

**Review by brian-learningpool — APPROVED** [review]

**Pull request closed without merging**